local/compose/logs.go

```diff
@@ -88,31 +88,41 @@ func (s *composeService) Logs(ctx context.Context, projectName string, consumer
 }
 
 type splitBuffer struct {
+	buffer    bytes.Buffer
 	service   string
 	container string
 	consumer  compose.ContainerEventListener
 }
 
 // getWriter creates a io.Writer that will actually split by line and format by LogConsumer
 func getWriter(service, container string, events compose.ContainerEventListener) io.Writer {
```

Comment:
I moved this method and struct to utils/logconsumer.go as I needed it in the kube backend too(to avoid having duplicated code). Not sure why this code was back here. 

And above we call utils.GetWriter.

Reply:
https://github.com/docker/compose-cli/issues/1311

Reply:
Should this method be called on line 78 above?